pages/glossary/element-language.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Element Language
```

Comment:
I'm not sure it's a good idea to have 3 definitions in 1
This definition of "element language" defines 3 concepts (and none is called element language):
- default language of an element
- default page element
- common language (of an element)

When I start reading a rule and find a reference to "common element language" pointing to this file, my first instinct is that it is a wrong reference because it takes me so long to find the definition.